crates/state-store/src/state_store.rs

```diff
@@ -201,9 +219,8 @@ impl StateStore for DynamoDbStateStore {
 
     async fn put_query(&self, query: Query) -> Result<()> {
         let mut item = HashMap::new();
-        let parsed_start_time = Self::parse_start_time(&query.start_time)?;
-        let pk = Self::query_pk(&parsed_start_time);
-        let sk = Self::query_sk(&parsed_start_time);
+        let pk = Self::query_pk(&query.start_time);
```

Comment:
while we have pk as start_time here? I assumed pk should be shorten to date for pk.